### ISSUE TITLE
Garbage characters are inserted if you come back from terminal output to input

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#105425]
CC @isidorn

## System Info

* JAWS Version 2020.2006.12.400
* NVDA Version 2020.2

Version: 1.49.0-insider (system setup)
Commit: 7df29b1236ecf8a21af9fe19a52923f6d69e0577
Date: 2020-08-26T07:58:02.868Z
Electron: 9.2.0
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Windows_NT x64 10.0.19041

## Steps to Reproduce:

1. Load `code-insiders .` with a screen reader (e.g., NVDA).
1. Press Ctrl+` to put your focus in terminal input field.
1. Press Shift+Tab to focus in terminal output area.
1. Press Tab to go back to the terminal input field.

Garbage characters are inserted if you come back from terminal output to input so you will have to clean up your input field each time you execute any commands.

## Does this issue occur when all extensions are disabled?:

Yes

**Comment by Tyriar** [comment]
@jooyoungseo does this still happen?

**Comment by meganrogge** [comment]
I cannot reproduce this using NVDA
